spec/data/approved_tags.json

```diff
@@ -0,0 +1,150 @@
+{
+    "approved_http_tags": [
+        "2FAServer",
+        "Accounting",
+        "ADC",
+        "Admin Panel",
+        "Administrative",
+        "Analytics",
+        "API",
+        "API Gateway",
+        "APIServer",
+        "APM",
+        "Appliance",
+        "ApplicationServer",
+        "ASGIServer",
+        "AuthenticationServer",
+        "AuthorizationServer",
+        "B2B",
+        "BaaS",
+        "Cache",
+        "Caching Proxy",
+        "CAS",
+        "CASB",
+        "CDN",
+        "CI/CD",
+        "Cloud",
+        "CMS",
+        "Collaboration",
+        "Communication",
+        "Compliance",
+        "COTS",
+        "CRM",
+        "Customer Service",
+        "DAM",
+        "Database",
+        "Development",
+        "DMS",
+        "DNSServer",
+        "Documentation",
+        "Domain Management",
+        "DRM",
+        "Drupal Plugin",
+        "E-Commerce",
+        "E-Learning",
+        "EbookServer",
+        "Editor",
+        "Education",
+        "Email",
+        "Email Encryption",
+        "Embedded",
+        "Engineering",
+        "Entertainment",
+        "Extension",
+        "Fax",
+        "Finance",
+        "Firewall",
+        "Firmware",
+        "Forum",
+        "Framework",
```

Comment:
Maybe too generic? Not sure

Reply:
Are you in favour of having more specific and detailed tags rather than catch all ones?